webapp/config.py

```diff
@@ -50,4 +50,5 @@ class ProdConfig(ConfigBase):
     'Prod': ProdConfig()
 }
 
-ENVAR_FINTECH_CONFIG = 'FINTECH_CONFIG'
\ No newline at end of file
+ENVAR_FINTECH_CONFIG = 'FINTECH_CONFIG'
+os.environ['FINTECH_CONFIG'] = 'Dev'
```

Comment:
Sir ,
It was done because we had to add the environment variable for Dev inorder to resolve the problem  theapp.config.from_object(config.configs[os.getenv(config.ENVAR_FINTECH_CONFIG)]).